weed/command/scaffold.go

```diff
@@ -353,9 +353,14 @@ directory = "/buckets"
 
 [sink.local]
```

Comment:
how about "local_incremental" and "local"?

Reply:
ok